Common/AR/Ar.md

```diff
@@ -0,0 +1,340 @@
+
+3. In Qt Create, create a new project and select `ArcGIS Runtime 100.6 Qt Quick QML app`.
+Select the option "3D project" in the "Details" step.
+
+4. In the created project, add these lignes in the `.pro` file. These lines can be anywhere.
+Set the correct path to the ArcGIS Toolkit source folder.
+```
+AR_TOOLKIT_SOURCE_PATH = # must be set to the path to toolkit sources
+include($$AR_TOOLKIT_SOURCE_PATH/Plugin/QmlApi/ArQmlApi.pri)
+```
+
+5. In the `main.cpp` file.
+
+5a. Set the environment variable `QSG_RENDER_LOOP` to `basic` in the begining of the `main` function.
+See [Note for performance issues] for details.
+```
+#if defined(Q_OS_IOS) || defined(Q_OS_ANDROID)
+  qputenv("QSG_RENDER_LOOP", "basic");
+#endif
+```
+
+5b. Include the header `QmlArcGISArView.h`.
+```
+#include "QmlArcGISArView.h"
+```
+
+5c. Call the `QmlArcGISArView::qmlRegisterTypes()` before the creation of the QML engine, to register the
+`ArcGISArView` and `LocationDataSource` QML types in QML engine. This is necessary to create a object
+of these types in QML.
+```
+Esri::ArcGISRuntime::Toolkit::QmlArcGISArView::qmlRegisterTypes();
+```
+
+6. In the `main.qml` file.
+
+6a. Import the AR toolkit:
+```
+import Esri.ArcGISArToolkit 1.0
+```
+
+6b. Create an AR view in the back of the `SceneView` (then before the declaration of `SceneView` in the QML file).
+The following code show an example of initialization for the properties `originCamera`, `translationFactor` and
+`locationDataSource`used for full scale mode.
+```
+ArcGISArView {
+    id: arcGISArView
+    anchors.fill: parent
+    tracking: true
+    sceneView: sceneView
+    originCamera: Camera {}
+    locationDataSource: LocationDataSource {}
+    translationFactor: 1.0
+}
+```
+
+The following code show an example for tabletop mode.
+```
+originCamera: Camera {
+    location: Point {
+        y: latitude
+        x: longitude
+        z: altitude
+    }
+    heading: 0.0
+    pitch: 90.0
+    roll: 0.0
+}
+locationDataSource: null
+translationFactor: 1000.0
+```
+
+## AR frameworks configuration
+
+All the files necessary to build an ArcGIS application with AR support are provided in the AR toolkit.
+To deploy the application on Android and iOS devices, extra steps are needed to give the permission
+to the application.
+
+### Configuration for iOS devices
+
+1. Open the `Info.plist` file and add the following key at the end of the file, just before `</dict>`,
+to give the access to the camera to the `ARKit`.
+```
+<key>NSCameraUsageDescription</key>
+<string>Camera access is needed for AR testing</string>
+```
+
+See [NSCameraUsageDescription Apple's documentation](https://developer.apple.com/documentation/bundleresources/information_property_list/nscamerausagedescription?language=objc) for details.
+
+2. The project is ready to be build and run in an iOS device. You need to verify the compatibility of your device
```

Comment:
Is there a link to any doc that explains which devices are compatible for iOS?

Reply:
```

2. The project is ready to be build and run in a compatible iOS device. AR requires an iOS or iPadOS device
with iOS 11 and an A9 processor or later. The list of compatible devices can be found in the end of the
[Augmented Reality page](https://www.apple.com/lae/ios/augmented-reality/)
```